src/utils.ts

```diff
@@ -108,23 +109,24 @@ export const getPaymentInfoForExtrinsic = async (
   const { signer } = extrinsic;
 
   let dispatchable = buildDispatchable(context, extrinsic);
-  let wallet = await getWallet(signer);
+  let wallet = await getWallet(signer, extrinsic.keyPairType);
 
   return await dispatchable.paymentInfo(wallet);
 };
 
 export const getWallet = async (
-  uri: string
+  uri: string,
+  type?: KeypairType
 ): Promise<
   | KeyringPair
   | {
       address: any;
       addressRaw: Uint8Array;
     }
 > => {
-  if (uri.substring(0, 2) === '//') {
+  if (uri.substring(0, 2) === '//' || uri.substring(0, 2) === '0x' ) {
```

Comment:
This was required to handle the Ethereum private key as a hex address, not sure of the effects on the `else` block below - i.e. does checking for 0x here break the `address / addressRaw` support below?